development/tools/jobs/testinfra/validate_test.go

```diff
@@ -13,16 +13,16 @@ func TestValidateProwPresubmit(t *testing.T) {
 	jobConfig, err := tester.ReadJobConfig("./../../../../prow/jobs/test-infra/validation.yaml")
 	// THEN
 	require.NoError(t, err)
-	testInfraPresubmits := jobConfig.Presubmits["kyma-project/test-infra"]
+	testInfraPresubmits := jobConfig.AllStaticPresubmits([]string{"kyma-project/test-infra"})
```

Comment:
Here you already use the `AllStaticPresubmits` where in all other test files you're accessing the field directly. I think the way with the method is cleaner.

Reply:
Got it changed in the latest commit.